data/openbangla-keyboard.desktop.in

```diff
@@ -1,7 +1,7 @@
 [Desktop Entry]
 Name=OpenBangla Keyboard
 Comment=An OpenSource, Unicode compliant Bengali Input Method.
-Exec=@PROJECT_DATADIR@/openbangla-gui %f
+Exec=@LIBEXECDIR@/openbangla-gui %f
```

Comment:
This will need to change as we're considering placing the binary in a different place.